i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/workloadspread.md

```diff
@@ -286,6 +286,65 @@ spec:
     ...
 ```
 
+### 支持 AI 负载
```

Comment:
the field "minReplicas" is used in the blog and other wechat docs but not documented, maybe we can add the description in ud and uniteddeployment